docs/src/index.md

```diff
@@ -414,38 +414,34 @@ more obvious with less noise from `@` symbols. This approach also
 avoids filling up the limited macro name space. The main downside is
 that more magic happens under the hood.
 
-Alternatively you can use Lazy.jl `@>` macro like this:
+Alternatively you can use Chain.jl, which exports the `@chain` macro. With Chain.jl,
+there is no need for `|>` and the first result of the previous expression is 
+assumed to be the first argument of the current call, unless otherwise specified 
+with `_`.
 
 ```julia
-using Lazy: @>
+using Chain, Statistics
 
 df = DataFrame(a = repeat(1:5, outer = 20),
                b = repeat(["a", "b", "c", "d"], inner = 25),
                x = repeat(1:20, inner = 5))
 
-x_thread = @> begin
-    df
-    @transform(y = 10 * :x)
-    @where(:a .> 2)
-    @by(:b, meanX = mean(:x), meanY = mean(:y))
-    @orderby(:meanX)
-    @select(:meanX, :meanY, var = :b)
+x_thread = @chain df begin
+    @transform(_, y = 10 * :x)
+    @where(_, :a .> 2)
+    @by(_, :b, meanX = mean(:x), meanY = mean(:y))
+    @orderby(_, :meanX)
+    @select(_, :meanX, :meanY, var = :b)
```

Comment:
I thought `_` as the first argument was implicit?

Reply:
yup, sorry about that. fixed.